install.sh

```diff
@@ -15,26 +15,26 @@ USAGE:
     install.sh [FLAGS] [OPTIONS]
 FLAGS:
     -h, --help               Prints help information
-        --force              Force reinstall all components if they are already installed, include: helm, kind, local-kube, chaos-mesh
+        --force              Force reinstall all components if they are already installed, include: kind, local-kube, chaos-mesh
         --force-chaos-mesh   Force reinstall chaos-mesh if it is already installed
         --force-local-kube   Force reinstall local Kubernetes cluster if it is already installed
         --force-kubectl      Force reinstall kubectl client if it is already installed
         --force-kind         Force reinstall Kind if it is already installed
-        --force-helm         Force reinstall Helm if it is already installed
-        --dashboard          Install Chaos Dashboard
         --docker-mirror      Use docker mirror to pull image, dockerhub.azk8s.cn => docker.io, gcr.azk8s.cn => gcr.io
         --volume-provisioner Deploy volume provisioner in local Kubernetes cluster
         --local-registry     Deploy local docker registry in local Kubernetes cluster
+        --dry-run            Simulate an install and generate chaos-mesh manifests files
 OPTIONS:
     -v, --version            Version of chaos-mesh, default value: latest
     -l, --local [kind]       Choose a way to run a local kubernetes cluster, supported value: kind,
                              If this value is not set and the Kubernetes is not installed, this script will exit with 1.
     -n, --name               Name of Kubernetes cluster, default value: kind
+    -c  --crd                The URL of the crd files, default value: https://raw.githubusercontent.com/pingcap/chaos-mesh/master/manifests/crd.yaml
```

Comment:
What is the use case that we need to expose this flag? Does it need to be configurable?

Reply:
Using a relative path is enough here?

Reply:
If we use a `relative path`, we must use exec this script in specific directory.

Reply:
Yes, but we have to. It is not appropriate to use `/master` in such script. If we want to run this script in the release branch, then what is the value of `crd`? Do we have to update to `https://raw.githubusercontent.com/pingcap/chaos-mesh/<tag>/manifests/crd.yaml` every time?

Reply:
When we need to release, we also need to change the image to install, so I think we also need to update this, or we add a flag to define the version of chaos-mesh.

Reply:
I still think it is better to use `relative path`. If using `master`, then how to test each PR in CI? We have to update the path to each PR's CRD?

Reply:
this pr has no relation to CI. Why need to use this pr in CI?

Reply:
Sorry I mean not CI, just in each branch. If we are in different branches and run this script, then we have to still update the crd url every time? I don't think it is reasonable

Reply:
Maybe we can support to set a `relative path` and use the URL as a default value is OK.

Reply:
Done